modules/ROOT/pages/manage-proxy.adoc

```diff
@@ -62,11 +51,22 @@ rtfctl apply http-proxy "http://<user>:<pass>@<10.0.0.1>:<8080>" --no-proxy "<1.
 The password must be URL encoded.
 * *<10.0.0.1>*: the IP address or hostname to access the HTTP proxy.
 * *<8080>*: the port on the host where the HTTP proxy is listening for requests.
-* *<1.1.1.1:8888,2.2.2.2.:9999>*: the `NO_PROXY` hosts and ports, delimited by commas.
+* *<1.1.1.1:8888,2.2.2.2:9999>*: the `NO_PROXY` hosts and ports, delimited by commas.
 . To verify the change was successful, run the following command to output the current value of the HTTP proxy: `rtfctl get http-proxy`.
 
 The output should match the value expected.
 
+=== Run the Script on Each Node
+
```

Comment:
Can we provide a bit of intro here to provide context and tell why or under what conditions the audience wants or needs to perform this task? Before/after what other tasks? Why?